omnibus/package-scripts/chef-server/postinst

```diff
@@ -11,6 +11,10 @@ function error_exit
 	exit 1
 }
 
+# If we have progressed from the preinst script
+# It implies that the user has agreed to proceed with the reindexing
+touch /tmp/first_time_internal_elasticsearch_install
```

Comment:
I think we should use a path in a folder we otherwise control, such as `/var/opt/opscode`